cmd/app/settings.go

```diff
@@ -0,0 +1,121 @@
+	"github.com/slackapi/slack-cli/internal/shared"
+	"github.com/slackapi/slack-cli/internal/slackerror"
+	"github.com/slackapi/slack-cli/internal/slacktrace"
+	"github.com/slackapi/slack-cli/internal/style"
+	"github.com/spf13/cobra"
+)
+
+var settingsAppSelectPromptFunc = prompts.AppSelectPrompt
+
+func NewSettingsCommand(clients *shared.ClientFactory) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "settings [flags]",
+		Short: "Open app settings for configurations",
+		Long: strings.Join([]string{
+			"Open app settings to configure an application in a web browser.",
+			"",
+			"Discovering new features and customizing an app manifest can be done from this",
+			fmt.Sprintf("intuitive interface for apps with a \"%s\" manifest source.", config.ManifestSourceRemote.String()),
+			"",
+			"This command does not support apps deployed to Run on Slack infrastructure.",
+		}, "\n"),
+		Example: style.ExampleCommandsf([]style.ExampleCommand{
+			{
+				Meaning: "Open app settings for a prompted app",
+				Command: "app settings",
+			},
+			{
+				Meaning: "Open app settings for a specific app",
+				Command: "app settings --app A0123456789",
+			},
+		}),
+		Args: cobra.MaximumNArgs(0),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return appSettingsCommandPreRunE(clients, cmd, args)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return appSettingsCommandRunE(clients, cmd, args)
+		},
+	}
+	return cmd
+}
+
+// appSettingsCommandPreRunE determines if the command can be run in a project
+func appSettingsCommandPreRunE(clients *shared.ClientFactory, cmd *cobra.Command, args []string) error {
+	ctx := cmd.Context()
+	err := cmdutil.IsValidProjectDirectory(clients)
+	if err != nil {
+		return err
+	}
+	if clients.Config.ForceFlag {
+		return nil
+	}
+	err = cmdutil.IsSlackHostedProject(ctx, clients)
+	if err != nil {
+		if slackerror.Is(err, slackerror.ErrAppNotHosted) {
+			return nil
+		} else {
+			return err
+		}
+	}
+	return slackerror.New(slackerror.ErrAppHosted).
+		WithDetails(slackerror.ErrorDetails{
+			{
+				Message: "App settings is not supported with Run on Slack infrastructure",
+			},
+		})
+}
+
+// appSettingsCommandRunE opens app settings in a browser for the selected app
+func appSettingsCommandRunE(clients *shared.ClientFactory, cmd *cobra.Command, args []string) error {
+	ctx := cmd.Context()
+	clients.IO.PrintTrace(ctx, slacktrace.AppSettingsStart)
+
+	app, err := settingsAppSelectPromptFunc(ctx, clients, prompts.ShowInstalledAndUninstalledApps)
+	if err != nil {
+		return err
+	}
+	host := clients.API().Host()
+	parsed, err := url.Parse(host)
+	if err != nil {
+		return err
+	}
+	parsed.Host = "api." + parsed.Host
+	settingsURL := fmt.Sprintf("%s/apps/%s", parsed.String(), app.App.AppID)
+
+	clients.IO.PrintInfo(ctx, false, "\n%s", style.Sectionf(style.TextSection{
+		Emoji: "house",
+		Text:  "App Settings",
+		Secondary: []string{
+			settingsURL,
+		},
```

Comment:
suggestion: What do you think about adding output explaining the action being taken (opening the web browser)?
```suggestion
		Secondary: []string{
			"Opening default web browser:",
			settingsURL,
		},
```

☝🏻 You might have better wording than that.

Reply:
🤔 I might have preference against this since erroring cases might cause confusion that the standalone link saves?

The balance of text also feels lopsided but I don't think this is significant reason against this:

```
🏠 App Settings
   Opening default web browser:
   https://api.slack.com/apps/A08SPJ83H4L
```

Reply:
📝 It does make me think `--no-browser` might be a flag to later support with the default behavior though!

Reply:
Sounds good to me. Also `--no-browser` would be an appropriate future flag, since `activity --browser` already exists to open the activity in the browser.